ibis/expr/operations/relations.py

```diff
@@ -311,7 +325,11 @@ class AnyInnerJoin(Join):
 
 @public
 class AnyLeftJoin(Join):
-    pass
+    @property
+    def schema(self) -> Schema:
+        return Schema(
+            {name: typ.copy(nullable=True) for name, typ in super().schema.items()}
+        )
```

Comment:
Turns that all of these are necessary to address the case of non-overlapping columns 😅